simapp/simd/main.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 func main() {
+	var _ = os.Args // To avoid linter "imported but not used" false positive
```

Comment:
I think we should create an issue.

Reply:
Yes, this linter problem should eventually be corrected. I registered this issue:
https://github.com/Finschia/ramus/issues/63